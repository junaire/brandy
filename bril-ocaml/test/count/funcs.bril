@main {
  a: int = const 2;
  b: int = id a;
  c: bool = eq a b;
  d: bool = id c;
}

@f(i: int): bool {
  zero: int = const 0;
  b: bool = eq i zero;
  ret b;
}
