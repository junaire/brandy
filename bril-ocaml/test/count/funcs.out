Ints: 3 Bools: 3
